languages/ruby/reference/exercise-concepts/flatten-array.md

```diff
@@ -0,0 +1,29 @@
+# Concepts of flatten-array
+
+## General
+
+- method definition: the student must implement the `flatten` method
+- method arguments (positional): `flatten` takes one argument
+- return values: `flatten` should return an array
+- class methods: one approach is the define a class `FlattenArray` with a class method `flatten`
+- module methods: one approach is to implement `FlattenArray` as a module
+- arrays: the input is an array, and student's should understand the basics of arrays
+- array flatten: this method is the obvious/preferred way to flatten an array
```

Comment:
I don't think this might not be a Concept, but an example of a "list transformation" Concept or something like that.

Reply:
Yeah maybe it's a bit too small to be a concept on it's own :/ Should have "list transformations" as a more comprehensive concept then, supported by practice exercises for specific common list transformations?